drivers/spi/spi_nrfx_spi.c

```diff
@@ -271,23 +272,49 @@ static int init_spi(struct device *dev, const nrfx_spi_config_t *config)
 	return 0;
 }
 
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+static int spi_nrfx_pm_control(struct device *dev, u32_t ctrl_command,
+				void *context, device_pm_cb cb, void *arg)
+{
+	static u32_t current_state = DEVICE_PM_ACTIVE_STATE;
```

Comment:
This cannot be a common variable for all instances, like it is now. Each instance needs to store its own state separately. Same applies to SPIM (and to TWI and TWIM that have already been merged, unfortunately).